openmls/src/group/tests/test_proposal_validation.rs

```diff
@@ -1062,11 +1062,11 @@ fn test_valsem104(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider
     }
 }
 
-/// ValSem105:
+/// ValSem113:
 /// Add Proposal:
-/// HPKE init key in proposals must be unique among existing group members
+/// HPKE init key and encryption key must be different
 #[apply(ciphersuites_and_backends)]
-fn test_valsem105(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider) {
+fn test_valsem113(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider) {
```

Comment:
Is there also a test for the new ValSem114? Or did I miss it? It would be nice to have at least a basic test.